.gitlab-ci.yml

```diff
@@ -158,156 +158,16 @@ test:acceptance:
-        if [ $? -eq 1 ]; then
-          return 0
-        fi
-        return 1
-      }
-
-    # Bash function to securely copy packages to the pool, warning and not copying
-    # if the files already exist
-    - |
-      function copy_if_non_existent() {
-        packages_glob="$1"
-        pool_dest=$2
-
-        first_filename=$(basename $(ls $packages_glob | head -1))
-        if [ -f ${pool_dest}/${first_filename} ]; then
-          echo "WARNING: $(basename "$packages_glob") exist in $pool_dest. Not overriding."
-        else
-          cp $packages_glob $pool_dest
-        fi
-      }
-
-    # For each package:
-    #  * if version is newer and is a final tag
-    #      --> include to stable via reprepro
-    #  * if version is newer but not final tag (master of build tags)
-    #      --> include to experimental via reprepro
-    #  * else version is older
-    #      --> manually copy to pool unprocessed (and unsigned) by reprepro
-
-    # mender-client
-    - if is_final_tag ${MENDER_VERSION}; then
-    -   codename=stable
-    - else
-    -   codename=experimental
-    - fi
-    - if is_newer_version mender-client ${MENDER_VERSION} ${codename}; then
-    -   for change_file in $(ls ${CI_PROJECT_DIR}/output/mender-client_*.changes); do
-    -     reprepro --keepunreferencedfiles include ${codename} ${change_file}
-    -   done
-    - else
-    -   copy_if_non_existent "${CI_PROJECT_DIR}/output/mender-client*.deb" pool/main/m/mender-client/
-    - fi
-
-    # mender-connect
-    - if is_final_tag ${MENDER_CONNECT_VERSION}; then
-    -   codename=stable
-    - else
-    -   codename=experimental
-    - fi
-    - if is_newer_version mender-connect ${MENDER_CONNECT_VERSION} ${codename}; then
-    -   for change_file in $(ls ${CI_PROJECT_DIR}/output/mender-connect_*.changes); do
-    -     reprepro --keepunreferencedfiles include ${codename} ${change_file}
-    -   done
-    - else
-    -   copy_if_non_existent "${CI_PROJECT_DIR}/output/mender-connect*.deb" pool/main/m/mender-connect/
-    - fi
-
-    # mender-configure
-    - if is_final_tag ${MENDER_CONFIGURE_VERSION}; then
-    -   codename=stable
-    - else
-    -   codename=experimental
-    - fi
-    - if is_newer_version mender-configure ${MENDER_CONFIGURE_VERSION} ${codename}; then
-    -   for change_file in $(ls ${CI_PROJECT_DIR}/output/mender-configure_*.changes); do
-    -     reprepro --keepunreferencedfiles include ${codename} ${change_file}
-    -   done
-    - else
-    -   copy_if_non_existent "${CI_PROJECT_DIR}/output/mender-configure*.deb" pool/main/m/mender-configure/
-    - fi
-
-    # Upload to bucket
-    - aws s3 sync db s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/db
-    - aws s3 sync --acl public-read pool s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/pool
-    - aws s3 sync --acl public-read dists s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/dists
+    # Upload .changes and .deb files
+    - for file in $(find output/ -name *.changes -o -name *.deb); do
```

Comment:
@lluiscampos please upload deb packages/buildinfo first, thenthen the changes file. Is there a particular reason why you are not uploading the buildinfo file?

Reply:
> please upload deb packages/buildinfo first, thenthen the changes file.

I can do that. As I am taking a lock before any copy I didn't think about any particular order

> Is there a particular reason why you are not uploading the buildinfo file?

No reason at all. I just realized that we don't do any use of them from the script in the VM, so decided to drop. I can add them if you want.

Reply:
The script on the VM processes all the files listed in `changes`, if you don't mind, I would upload them all, leaving the `changes` file last.